trycourier.gemspec

```
@@ -1,15 +1,15 @@
-require_relative 'lib/trycourier/version'
+require_relative "lib/trycourier/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "trycourier"
-  spec.version       = Courier::VERSION
-  spec.authors       = ["Courier"]
-  spec.email         = ["support@trycourier.com"]
+  spec.name = "trycourier"
+  spec.version = Courier::VERSION
+  spec.authors = ["Courier"]
+  spec.email = ["support@trycourier.com"]
```

We need to make sure to get rid of all the trycourier urls (trycourier.app, trycourier.com)
```suggestion
spec.email = ["support@courier.com"]
```